streams/examples/main/java/org/apache/kafka/streams/examples/pageview/PageViewUnTypedJob.java

```diff
@@ -31,27 +32,41 @@
 import org.apache.kafka.streams.KafkaStreams;
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.StreamsConfig;
-import org.apache.kafka.streams.kstream.Aggregator;
 import org.apache.kafka.streams.kstream.HoppingWindows;
 import org.apache.kafka.streams.kstream.KStreamBuilder;
 import org.apache.kafka.streams.kstream.KStream;
 import org.apache.kafka.streams.kstream.KTable;
 import org.apache.kafka.streams.kstream.KeyValueMapper;
+import org.apache.kafka.streams.kstream.ValueJoiner;
+import org.apache.kafka.streams.kstream.ValueMapper;
 import org.apache.kafka.streams.kstream.Windowed;
 
 import java.util.Properties;
 
-public class PageViewUnTypedJob {
+/**
+ * Implements the "PageView Stats" program that joins the pageview stream with
+ * the user profile table to count the number of pageview per user region.
+ * Assumes the input stream is named "streams-pageview-input".
+ *
+ * This example shows how to:
+ *   1. write stateful computations like joins / aggregations using the high-level KStream DSL.
+ *   2. use general data types (e.g. JSON, Avro, etc) for serdes in Kafka Streams.
```

Comment:
I'd clarify to sth like:

>  2) use general data types (here: JSON;  but can also be Avro generic bindings, etc.) for serdes in Kafka Streams.

To make it clear that this example does not showcase Avro usage.